src/main/java/models/Address.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * COPYRIGHT: FREQUENTIS AG. All rights reserved.
+ *            Registered with Commercial Court Vienna,
+ *            reg.no. FN 72.115b.
+ */
+package models;
+
+import com.sun.istack.internal.NotNull;
```

Comment:
`internal` or `sun` packages are recommended not to be used. They are subject to change without notice in future JDK releases and it could break our code.